parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/ParallelEoSStreamProcessor.java

```diff
@@ -48,8 +50,7 @@ public void poll(Consumer<PollContext<K, V>> usersVoidConsumptionFunction) {
 
     @Override
     @SneakyThrows
-    public void pollAndProduceMany(Function<PollContext<K, V>, List<ProducerRecord<K, V>>> userFunction,
-                                   Consumer<ConsumeProduceResult<K, V, K, V>> callback) {
+    public void pollAndProduceMany(Function<PollContext<K, V>, List<ProducerRecord<K, V>>> userFunction, Consumer<ConsumeProduceResult<K, V, K, V>> callback) {
```

Comment:
```suggestion
    public void pollAndProduceMany(Function<PollContext<K, V>, List<ProducerRecord<K, V>>> userFunction, 
        Consumer<ConsumeProduceResult<K, V, K, V>> callback) {
```
format with the line break, see if it lines up again, or is removed?